blueprints/secops/sops/ops2deb.yml

```diff
@@ -0,0 +1,18 @@
+name: sops
+matrix:
+  versions:
+    - 3.8.0
+    - 3.7.3
+    - 3.7.2
+  architectures:
+    - amd64
+    - arm64
+homepage: https://age-encryption.org/
+summary: Simple and flexible tool for managing secrets
+description: |-
+  sops is an editor of encrypted files that supports YAML, JSON, ENV, INI and
+  BINARY formats
+  and encrypts with AWS KMS, GCP KMS, Azure Key Vault, age, and PGP.
+fetch: https://github.com/getsops/sops/releases/download/v{{version}}/sops-v{{version}}.linux.{{goarch}}
+script:
```

Comment:
When possible, use install instead of script:

```yaml
install:
  - sops-v{{version}}.linux.{{goarch}}:/usr/bin/sops
```

Reply:
thx fixed